google/devtools/clouderrorreporting/v1beta1/clouderrorreporting_gapic.yaml

```diff
@@ -0,0 +1,177 @@
+type: com.google.api.codegen.ConfigProto
+generate_samples: true
+language_settings:
+  java:
+    package_name: com.google.cloud.devtools.clouderrorreporting.spi.v1beta1
+  python:
+    package_name: google.cloud.devtools.clouderrorreporting.v1beta1
+  go:
+    package_name: cloud.google.com/go/devtools/clouderrorreporting/apiv1beta1
+  csharp:
+    package_name: Google.Devtools.Clouderrorreporting.V1beta1
+  ruby:
+    package_name: Google::Devtools::Clouderrorreporting::V1beta1
+  php:
+    package_name: Google\Cloud\Devtools\Clouderrorreporting\V1beta1
```

Comment:
Done. Jun, can you modify the config generator in `toolkit` to produce this line, or file an issue to do so?